scripts/BorderControl.js

```diff
@@ -1,703 +1,764 @@
-    el.find('[name="Border-Control.controlledColor"]').parent().append(`<input type="color"value="${cC}" data-edit="Border-Control.controlledColor">`)
-    el.find('[name="Border-Control.partyColor"]').parent().append(`<input type="color"value="${pC}" data-edit="Border-Control.partyColor">`)
-    el.find('[name="Border-Control.targetColor"]').parent().append(`<input type="color"value="${tC}" data-edit="Border-Control.targetColor">`)
-
-    el.find('[name="Border-Control.neutralColorEx"]').parent().append(`<input type="color" value="${nCE}" data-edit="Border-Control.neutralColorEx">`)
-    el.find('[name="Border-Control.friendlyColorEx"]').parent().append(`<input type="color" value="${fCE}" data-edit="Border-Control.friendlyColorEx">`)
-    el.find('[name="Border-Control.hostileColorEx"]').parent().append(`<input type="color" value="${hCE}" data-edit="Border-Control.hostileColorEx">`)
-    el.find('[name="Border-Control.controlledColorEx"]').parent().append(`<input type="color"value="${cCE}" data-edit="Border-Control.controlledColorEx">`)
-    el.find('[name="Border-Control.partyColorEx"]').parent().append(`<input type="color"value="${pCE}" data-edit="Border-Control.partyColorEx">`)
-    el.find('[name="Border-Control.targetColorEx"]').parent().append(`<input type="color"value="${tCE}" data-edit="Border-Control.targetColorEx">`)
-
-    el.find('[name="Border-Control.healthGradientA"]').parent().append(`<input type="color"value="${gS}" data-edit="Border-Control.healthGradientA">`)
-    el.find('[name="Border-Control.healthGradientB"]').parent().append(`<input type="color"value="${gE}" data-edit="Border-Control.healthGradientB">`)
-    el.find('[name="Border-Control.healthGradientC"]').parent().append(`<input type="color"value="${gT}" data-edit="Border-Control.healthGradientC">`)
-
+  let nC = game.settings.get('Border-Control', 'neutralColor');
+  let fC = game.settings.get('Border-Control', 'friendlyColor');
+  let hC = game.settings.get('Border-Control', 'hostileColor');
+  let cC = game.settings.get('Border-Control', 'controlledColor');
+  let pC = game.settings.get('Border-Control', 'partyColor');
+  let nCE = game.settings.get('Border-Control', 'neutralColorEx');
+  let fCE = game.settings.get('Border-Control', 'friendlyColorEx');
+  let hCE = game.settings.get('Border-Control', 'hostileColorEx');
+  let cCE = game.settings.get('Border-Control', 'controlledColorEx');
+  let pCE = game.settings.get('Border-Control', 'partyColorEx');
+  let tC = game.settings.get('Border-Control', 'targetColor');
+  let tCE = game.settings.get('Border-Control', 'targetColorEx');
+  let gS = game.settings.get('Border-Control', 'healthGradientA');
+  let gE = game.settings.get('Border-Control', 'healthGradientB');
+  let gT = game.settings.get('Border-Control', 'healthGradientC');
+  el.find('[name="Border-Control.neutralColor"]')
```

Comment:
After seeing this I retract my statment about the core method being a bit uglier, the same ammount of code is required to reach the same result, with a less usefull and more obtuse UI

Reply:
I will revert to the old code